### DISCUSSION TITLE
Awww - it workeed and now it doesn't - HELP!!

### DISCUSSION THREAD

**Discussion by mintgitb1ker** [streamdeck-linux-gui/streamdeck-linux-gui#230] [Q&A] [answered]
Last week my Streamdeck worked with UI, lost my SSD and had to do a reinstall of everything. (Mint 21.3 Cinnamon - not wayland)
Now I get the following:
Streamdeck-ui recognizes Streamdeck original (I have V2)
It wont import my exported profile (states iot is for V2 not V1)
I can;t program the buttons as when I click to select the button it momenterrily highlights but deselects immediately (terminal shows the following when I try this)
```
Traceback (most recent call last):
  File "/home/rmint/.local/lib/python3.10/site-packages/streamdeck_ui/gui.py", line 523, in <lambda>
    button.clicked.connect(lambda button=button, buttons=buttons: button_clicked(ui, button, buttons))
  File "/home/rmint/.local/lib/python3.10/site-packages/streamdeck_ui/gui.py", line 407, in button_clicked
    button_id = selected_button.index  # type: ignore # Index property added
AttributeError: 'bool' object has no attribute 'index'
```

I loved streamdeck-ui when it worked

**Comment by coolapso** [comment]
Hi, have you read this one? https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203 

From that stack trace I can't really understand what's going on. but not sure if there will be anyone around to help you with it. Please consider changing to streamcontroller

**Reply by mintgitb1ker** [reply]
Thx. I did try that before streamdeck-ui - found it didn't work for me/was way more complicated (I only want to d simple things with my steamdeck!)
Maybe I should try it again
should I uninstall this one 1st? How?
(btw I am a bit old to say the least!!)

**Reply by mintgitb1ker** [reply]
Went back to Streamcntroller and starting to get results from that
Thanks so much
But Streamcontroller is sooo much more complicated - cant even work out how to add a simple keystroke like CTRL R
Tried unistalling streamdeck-ui via software manager and via rightclick uninatall but it still starts uo - its not even in Startup app list

**Reply by coolapso** [reply]
I think what's more confusing in the streamcontroller is that you have to go to the store and install the "deck" plugin to have the basic functionality you wish for, without it there's nothing you can do with the deck. 

This plugin should IMHO be installed by default, but I think there are plans for that. Looping @Core447 here so he can know, that has been one major pain point for users to make the switch. 

Also Feel free to join streamcontroller Discord, there;s a bunch of really active and helpful people there to help you with what you may need. 

The problem with the streamdeck application is that the code is too messy and its pretty much unmaintainable,  has no support for plugins and the effort for such a thing would be too big, as well as to do anything else on it. 

Initially I forked the project just to keep it running due to the lack of better alternative. My knowledge on python, is not good enough to maintain this application in its current state, nor I have the desire to go deeper with python.

**Reply by coolapso** [reply]
To uninstall try `pip uninstall streamdeck-linux-gui` or `pipx uninstall streamdeck-linux-gui` (assuming you installed it from the pip package manager)

**Reply by Core447** [reply]
Since 2 versions or so, there's a plugin screen in the onboarding dialog which installs the most important plugins by default.

**Comment by dhtseany** [comment]
Hello, can you please elaborate on the specific steps you took to install
the app?

I'm around today and I can attempt to provide assistance.

**Comment by mintgitb1ker** [comment]
Thanks so much
I used instructions from here:
https://timothycrosley.github.io/streamdeck-ui/docs/installation/ubuntu/

**Reply by coolapso** [reply]
Then you used the wrong documentation. that is from the Original project which was abandoned, hence this fork. 

the correct documentation for this project is here: https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/

So most likely you installed the old streamdeck-ui and not actually the streamdeck-linux-gui

**Reply by mintgitb1ker** [reply]
ah. silly me.
Any idea how I can get it off my computer not I have gone over to streamcontroller?

**Reply by coolapso** [reply, marked as answer]
`pip uninstall streamdeck-ui` should do the trick
